pages/properties/static.properties.json

```diff
@@ -1,6 +1,5 @@
 {
     "name": "Static HTML",
-    "creator": "GitHub Actions",
     "description": "Deploy static files in a repository without a build.",
     "iconName": "html",
     "categories": ["Pages"]
```

Comment:
doesn't this require language category?

Reply:
I can add `HTML`, `JavaScript`, `CSS`. Anything else you want to see here?

Reply:
It would be good to add those